clean.py

```diff
@@ -69,7 +68,11 @@ def quick_size_estimation(ds, content_key="text"):
     indices = rng.choice(len(ds), size=subset_size, replace=False, shuffle=False)
     partial_ds = ds.select(indices)
     ratio = float(len(ds)) / float(subset_size)
-    return sys.getsizeof("".join(partial_ds[content_key])) * ratio
+
+    len_bytes = 0
+    for content in partial_ds[content_key]:
+        len_bytes += len(content.encode())
```

Comment:
Hopefully that's not too slow. 10000 documents can be long, so maybe `.map`?

Reply:
Good catch, let me see if I can quickly adapt it to use a `.map`

Reply:
Done, do you want to have a second look?